### DISCUSSION TITLE
.NET SDK for Pocketbase?

### DISCUSSION THREAD

**Discussion by Qws** [pocketbase/pocketbase#1049] [Q&A]
I have a .NET MAUI app and I would like to use PocketBase as backend.

For desktop apps I just like to use .NET (MAUI, Xamarin or WinUI) instead of Flutter or webapps.

**Comment by ganigeorgiev** [comment]
There is no official C#/.NET SDK.

Another user recently posted this - https://github.com/PRCV1/pocketbase-csharp-sdk, but I haven't tested it and cannot provide support for it.

Alternatively, depending on what services you are planning to use, you can also create a small wrapper around the Records REST API - https://pocketbase.io/docs/api-records/.

---

_Please note that sometime later this week there will be a stable v0.8.0 which comes with some breaking changes related to the users management (for a full list of changes for now you can find in the [v0.8.0-rc1 changelog](https://github.com/pocketbase/pocketbase/releases/tag/v0.8.0-rc1))._

**Reply by benallfree** [reply]
Added to https://github.com/benallfree/awesome-pocketbase :)
